src/svc/invocable-scripts/paths/scripts_scriptID.yml

```diff
@@ -4,33 +4,73 @@ get:
     - Data I/O endpoints
     - Invokable Scripts
   summary: Retrieve a script
-  description: Uses script ID to retrieve details of an invokable script.
+  description: |
+    Retrieve an invokable script.
+
+    #### Related Guides
+
+    - [Invoke custom scripts](https://docs.influxdata.com/influxdb/cloud/api-guide/api-invokable-scripts/)
   parameters:
     - in: path
       name: scriptID
       schema:
         type: string
       required: true
-      description: The script ID.
+      description: |
+        Script ID.
+        Only returns the script with this ID.
   responses:
     '200':
-      description: The requested script object.
+      description: |
+        Success.
+        The response body contains the requested script object.
       content:
         application/json:
           schema:
             $ref: '../schemas/Script.yml'
+          examples:
+            $ref: '../schemas/ScriptGetResponse.yml'
+    '404':
+      description: |
+        Not found.
+      content:
+        application/json:
+          schema:
+            $ref: "../../../common/schemas/Error.yml"
+          examples:
+            notFound:
+              summary: |
+                The requested script was not found.
+              value: {
+                "code":"not found",
+                "message":"script \"09afa3b220fe400\" not found"
+              }
     default:
-      description: Unexpected error
+      description: Internal server error.
       $ref: '../../../common/responses/ServerError.yml'
 patch:
   operationId: PatchScriptsID
   tags:
     - Invokable Scripts
   summary: Update a script
   description: |
-    Updates properties (`name`, `description`, and `script`) of an invokable script.
+    Updates an invokable script.
+
+    Use this endpoint to update the properties (`description`, and `script`)
```

Comment:
`name` is removed?

Reply:
that is correct. I tested it a few times and `name` cannot be updated.